go.test.sh

```diff
@@ -6,7 +6,7 @@ cat /dev/null > coverage.txt
 for dir in $(go list -f '{{.ImportPath}}={{join .TestGoFiles ","}}' ./...);do
     array=(${dir//=/ })
     if [ -n "${array[1]}" ]; then
-        go test -short -v -coverprofile=coverage.out -covermode=count "${array[0]}"
+        go test -gcflags=all=-l -short -v -coverprofile=coverage.out -covermode=count "${array[0]}"
```

Comment:
It's required by [gomonkey](https://github.com/agiledragon/gomonkey?tab=readme-ov-file#test-method)